hugegraph-ml/src/hugegraph_ml/data/hugegraph2dgl.py

```diff
@@ -0,0 +1,83 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import dgl
+import torch
+from pyhugegraph.api.gremlin import GremlinManager
+from pyhugegraph.client import PyHugeClient
+
+
+class HugeGraph2DGL:
+    def __init__(
+            self,
+            ip='127.0.0.1',
+            port="8080",
+            graph='hugegraph',
+            user='admin',
+            pwd='xxx'
+    ):
+        self._client: PyHugeClient = PyHugeClient(
+            ip=ip,
+            port=port,
+            graph=graph,
+            user=user,
+            pwd=pwd
+        )
```

Comment:
Lack optional `graphspace` param here (python-client has upgraded to `1.5.0`)

Should update the input param